adam-python/bdgenomics/adam/rdd.py

```diff
@@ -43,6 +43,42 @@ def __init__(self, jvmRdd, sc):
         self.sc = sc
 
 
+    def cache(self):
+        """
+        Caches underlying RDD in memory.
+
+        :return: Returns a new, cached RDD.
+        """
+
+        return self._replaceRdd(self._jvmRdd.cache())
+
+
+    def persist(self, sl):
+        """
+        Persists underlying RDD in memory.
```

Comment:
Persists underlying RDD in memory. &rarr; Persists underlying RDD in memory or disk.